.github/workflows/pre-release-workflow.yml

```diff
@@ -100,7 +113,7 @@ jobs:
       - name: Swazzler Release
         run: |
           cd ./embrace-swazzler3
-          ./gradlew clean release -Dorg.gradle.parallel=false -Prelease.useAutomaticVersion=true -Prelease.releaseVersion=${{ github.event.inputs.current_version }} -Prelease.newVersion=${{ github.event.inputs.next_version }}-SNAPSHOT --stacktrace
+          ./gradlew clean release -Dorg.gradle.parallel=false -Prelease.useAutomaticVersion=true -Prelease.pushReleaseVersionBranch=release/${{ github.event.inputs.version_to_release }} -Prelease.releaseVersion=${{ github.event.inputs.version_to_release }} -Prelease.newVersion=${{ github.event.inputs.next_version }}-SNAPSHOT --stacktrace
```

Comment:
This is where I modified the command to create a release branch in Swazzler. I followed the docs for the plugin, but I don't know if it actually works, so we'll have to do by trying it out